ajo/PersonalizationPayloads.md

```diff
@@ -0,0 +1,396 @@
+## Personalization Payloads
+
+### Proposition Fetch
+
+```http request
+curl --location 'https://edge.adobedc.net/ee/v1/interact?configId=0814ac07-ffeb-44c4-8633-85301d5e721c' \
```

Comment:
The formatting looks bad in the markdown document, please fix.

![screenshot-2023-09-26 at 11 51 27@2x](https://github.com/adobe/alloy-samples/assets/427213/ff33c177-7a46-48da-9915-71419aa9c95e)